client/template.go

```diff
@@ -58,7 +58,7 @@ type Template struct {
 	TerragruntVersion    string           `json:"terragruntVersion,omitempty"`
 	IsDeleted            bool             `json:"isDeleted,omitempty"`
 	BitbucketClientKey   string           `json:"bitbucketClientKey"`
-	IsGitHubEnterprise   bool             `json:"isGitHubEnterprise"`
+	IsGithubEnterprise   bool             `json:"isGitHubEnterprise"`
```

Comment:
readResourceData and writeResourceData would have converted this to "is_git_hub_..."
This change converts it to "is_github_..." which is how it's defined in the schema.